demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/CarReviewsDetailNewCarPriceQuotePage.java

```diff
@@ -0,0 +1,226 @@
+        keyword.checkDisplay(keyword.findElement(txtDescriptionTitle)) &&  keyword.checkDisplay(keyword.findElement(txtDescriptionSubtitle)) &&
+        keyword.checkClickable(keyword.findElement(txtChangZipcode)) &&  keyword.checkDisplay(keyword.findElement(txtYearMakeModel)) && 
+        keyword.checkClickable(keyword.findElement(txtChangVehicle)) &&  keyword.checkClickable(keyword.findElement(btnGetMyFreeQuote));
+        //&&  keyword.checkDisplay(keyword.findElement(txtStyle));// && checkSelect(checkboxDealer1) 
+        // && checkSelect(checkboxDealer2) && checkSelect(checkboxDealer3)
+    }
+
+    /**
+     * Click button Get My Free Quote
+     */
+    public void clickButtonGetMyFreeQuote() {
+        actionExitAds();
+        keyword.waitForElementIsClickable(keyword.findElement(btnGetMyFreeQuote)).click();
+        //keyword.click(keyword.findElement(btnGetMyFreeQuote));
+    }
+
+    /**
+     * Check Check car price quote section work correct when fill data and click button get my free quote
+     * @return
+     */
+    public boolean checkCarPriceQuoteSectionWorkCorrectWhenInputValidDataAndClickButtonGetMyFreeQuote() {
+        return keyword.checkDisplay(keyword.findElement(txtDescriptionTitle)) && 
+        keyword.checkDisplay(keyword.findElement(txtDescriptionSubtitle)) &&
+        keyword.checkDisplay(keyword.findElement(txtFirstName)) &&  keyword.checkDisplay(keyword.findElement(txtLastName)) && 
+        keyword.checkDisplay(keyword.findElement(txtEmail)) && keyword.checkDisplay(keyword.findElement(txtPhone)) &&  
+        keyword.checkClickable(keyword.findElement(btnSend)) && keyword.checkClickable(keyword.findElement(linkPolicy));
+    }
+
+    /**
+     * Check UI New Car You May Like correct
+     * @return
+     */
+    public boolean checkUINewCarYouMayLike() {
+        keyword.scrollToElement(keyword.findElement(txtTitleCarYouMayLike));
+        System.out.println(keyword.waitForElementIsDisplay(keyword.findElement(txtDescriptionSubtitleCYML)).getText());
+        return keyword.isMessageCorrect(keyword.findElement(txtDescriptionSubtitleCYML), descriptionSubtitleCYMLExpected) &&
+        checkNumberVihicleCard() && keyword.checkClickable(keyword.findElement(linkBrowseAllListings)) &&
+        keyword. checkDisplay(keyword.findElement(linkBrowseAllListings));
+    }
+
+    /**
+     * Check list card vihicle
+     * @return
+     */
+    public boolean checkNumberVihicleCard() {
+        List<WebElement> listCard = driver.findElements(txtVihicleCard);
+        return listCard.size()==12;
+    }
+
+    /**
+     * Click on card vihicle and press ESC
+     */
+    public void clickCardvihicleESC() throws InterruptedException {
+        keyword.scrollToElement(keyword.findElement(txtTitleCarYouMayLike));
+        keyword.click(keyword.findElement(txtVihicleCard1));
+        // Actions action = new Actions(driver);
+        // Thread.sleep(2000);
+	    // action.sendKeys(Keys.ESCAPE);
```

Comment:
If you don't you these code, I think you should delete it, don't comment like that

Reply:
This place is not finished yet, I comment to fix later. Thank you was review my pull requets